### PR TITLE
GROW-710 BorrowerCountry wrapper component

### PR DESCRIPTION
**Pull request by mlibazisi**

### CODE REVIEW

File: src/components/BorrowerProfile/BorrowerCountry.vue [hunk 1]
```diff
@@ -0,0 +1,94 @@
+<template>
+	<article>
+		<kv-map
+			class="
+				tw-rounded
+				tw-overflow-hidden
+				tw-my-5
```
Comment (mcstover) [inline src/components/BorrowerProfile/BorrowerCountry.vue:7]: Great job getting this component in place @mlibazisi. The only thing I updated in this file was to change the margin to only vertical and bump it up once more.

File: .storybook/stories/KvMap.stories.js [hunk 2]
```diff
@@ -5,19 +5,32 @@ export default {
 	component: KvMap,
 	args: {
 		autoZoomDelay: 1000,
+		height: null,
 		initialZoom: null,
 		lat: 37.700091,
 		long: -123.013243,
 		mapId: 0,
 		useLeaflet: false,
+		width: null,
```
Comment (mcstover) [inline .storybook/stories/KvMap.stories.js:14]: Added the width and height props here and in the template to fix up the stories!

File: src/components/Kv/KvMap.vue [hunk 3]
```diff
@@ -86,14 +86,14 @@ export default {
 		 */
 		lat: {
 			type: Number,
-			default: 37.700091
+			default: null
```
Comment (mcstover) [inline src/components/Kv/KvMap.vue:89]: Having defaults in lat + long was causing them to get set during the component creation and not updated when the props came in. Added the watcher and ability to initialize using those or on mounted.

**Review by emuvente — APPROVED** [review]
👍

**Pull request merged**